client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
```

Comment:
Yep yep, I asked just to be sure. It's not ideal, but it's better than using `!important`.